lib/pullrequest-decorator-reporter.js

```diff
@@ -30,6 +30,7 @@ const buildOptions = (reporterOptions, newmanOptions) => {
     showOnlyFails: reporterOptions.showOnlyFails,
     collectionName: newmanOptions.collection.name,
     debug: reporterOptions.debug,
+    enablePullRequestComment: reporterOptions.pullrequestEnableComment,
```

Comment:
It should be named reporterOptions.pullrequestDecoratorEnableComment, this way the parameter will be passed as --reporter-pullrequest-decorator-enableComment, look at the example here : 
<img width="776" alt="Screenshot 2023-10-23 at 12 14 12" src="https://github.com/Decathlon/newman-reporter-pullrequest-decorator/assets/45691655/49d01ae5-ab66-45f7-8344-a5efa98a914a">

Reply:
Done